### ISSUE TITLE
Go to new _www after rebuild

### ISSUE THREAD

**Issue by GoZOo** [platformsh/platformsh-cli#468]
If you launch a platform build when you are in _www (or subdirectory), you are in a nowhere path after the build since symlink has been removed and re-created. So you have to force going into the new directory (which have same path).

I suggest to launch `cd {current_path}` at the end of build, so we can automatically play drush commands or other commands without manually going back to _www.

I guess we have the same issue with build/default/*.

Potential issue with this new feature for drupal, when we are in a module directory which is no longer in project.make, rebuild will not generate this directory, so `cd path/to/module` will fail.

**Comment by pjcdawkins** [comment]
PHP processes can't do `cd {current_path}` themselves, but yes, the CLI could suggest doing `cd` after `platform build`.

Meanwhile, a shortcut to go to the project directory is:

```sh
cd "$(platform dir)"
```

or to go to the `_www` directory:

```sh
cd "$(platform dir web)"
```

or to run Drush in the `_www` directory:

```sh
pldr
```

**Comment by GoZOo** [comment]
Right, i don't look at code and i thought there was a little bit bash script in there.

Thanks for your answer Patrick
